tests/integration_tests/test_regularizers.py

```diff
@@ -68,66 +53,52 @@ def test_regularizers(
     torch.manual_seed(RANDOM_SEED)
     random.seed(0)
 
-    data_file = generate_data(
-        input_features,
-        output_features,
-        num_examples=BATCH_SIZE
-    )
+    data_file = generate_data(input_features, output_features, num_examples=BATCH_SIZE)
     data_df = read_csv(data_file)
 
     regularizer_losses = []
-    for regularization_type in [None, 'l1', 'l2', 'l1_l2']:
+    for regularization_type in [None, "l1", "l2", "l1_l2"]:
 
         config = {
-            'input_features': input_features,
-            'output_features': output_features,
-            'combiner': {'type': 'concat', 'fc_size': 14},
-            'training': {
-                'epochs': 2,
-                'regularization_type': regularization_type,
-                'regularization_lambda': 0.1
-            }
+            "input_features": input_features,
+            "output_features": output_features,
+            "combiner": {"type": "concat", "fc_size": 14},
+            "training": {"epochs": 2, "regularization_type": regularization_type, "regularization_lambda": 0.1},
         }
 
         backend = LocalTestBackend()
         model = LudwigModel(config, backend=backend)
-        processed_data_df, _, _, _ = preprocess_for_training(
-            config, data_df, backend=backend
-        )
+        processed_data_df, _, _, _ = preprocess_for_training(config, data_df, backend=backend)
         with processed_data_df.initialize_batcher(batch_size=BATCH_SIZE) as batcher:
             batch = batcher.next_batch()
 
         _, _, _ = model.train(
-                training_set=data_df,
-                skip_save_processed_input=True,
-                skip_save_progress=True,
-                skip_save_unprocessed_output=True
+            training_set=data_df,
+            skip_save_processed_input=True,
+            skip_save_progress=True,
+            skip_save_unprocessed_output=True,
         )
 
         inputs = {
-            i_feat.feature_name: batch[i_feat.proc_column]
+            i_feat.feature_name: torch.from_numpy(batch[i_feat.proc_column]).to(DEVICE)
             for i_feat in model.model.input_features.values()
         }
         targets = {
-            o_feat.feature_name: batch[o_feat.proc_column]
+            o_feat.feature_name: torch.from_numpy(batch[o_feat.proc_column]).to(DEVICE)
             for o_feat in model.model.output_features.values()
         }
         predictions = model.model((inputs, targets))
 
         loss, _ = model.model.train_loss(targets, predictions, regularization_type, 0.1)
         regularizer_losses.append(loss)
 
-    print(regularizer_losses)
-
     # Regularizer_type=None has lowest regularizer loss
     assert min(regularizer_losses) == regularizer_losses[0]
 
     # l1, l2 and l1_l2 should be greater than zero
-    assert np.all([t - regularizer_losses[0] > 0.0 for t in regularizer_losses[1:]])
+    assert torch.all(torch.tensor([t - regularizer_losses[0] > 0.0 for t in regularizer_losses[1:]]))
```

Comment:
I thought we did this already?
https://github.com/ludwig-ai/ludwig/pull/1553/files

Reply:
@anmshkmr Agreed! I think at some point (possibly because of a rebase or a force push) we lost those changes.

I looked through the [commit history](https://github.com/ludwig-ai/ludwig/pull/1508/commits) of the `torch-gpu` branch and couldn't find #1553 in that list either.